lib/site.js

```diff
@@ -106,18 +95,15 @@ module.exports = class Site {
 		}
 	}
 
-	// generates a URI for a given slug or `/`-delimited sequence thereof
-	pageURI(slugs) {
-		return path.join(this.baseURI, `${slugs}.html`); // TODO: validate slugs?
-	}
-
-	assetURI(filepath) {
-		return path.join(this.baseURI, this.assetPath, filepath); // TODO: validate?
-	}
```

Comment:
Where are URIs being generated now? I've noticed `Navigation` concatenates `baseURI` and `slug`, so the lack of encapsulation makes me uncomfortable once more (based on actual real-world experience).

Having said that: It's probably not important right now; we'll deal with it if and when it comes up during actual use.

Reply:
Yep, I need to think about that again. I think the current solution is a little fiddly.